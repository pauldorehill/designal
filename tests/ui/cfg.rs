use designal::Designal;

#[derive(Designal)]
#[designal(trim_start = "Human")]
struct HumanBean {
    #[designal(cfg_feature = "client")]
    taste: String,
}

fn main() {}