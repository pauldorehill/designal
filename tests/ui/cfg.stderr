error: You can't use cfg_feature on a field
 --> $DIR/cfg.rs:6:16
  |
6 |     #[designal(cfg_feature = "client")]
  |                ^^^^^^^^^^^
